backend/curriculum_tracking/serializers.py

```diff
@@ -928,3 +956,12 @@ def get_reviewer_user_emails(self, instance):
     def get_status(self, instance):
         if instance.agile_card:
             return instance.agile_card.status
+
+    def get_number_of_times_card_moved_to_review_feedback(self, instance):
+        number_of_times_card_moved_to_review_feedback = len(
+            LogEntry.objects.filter(effected_user=instance.recruit_users.first(),
```

Comment:
please avoid copy paste. DRY code is better.

You can add a function to the RecruitProject model. Then just call the function as needed.

There is no reason to introduce this technical debt now.